### ISSUE TITLE
Incorrect error message when updating the schema

### ISSUE THREAD

**Issue by erikvullings** [weaviate/typescript-client#73]
When updating the schema, the error message swaps the class and property name, e.g. 

```bash
Error: usage error (422): {"error":[{"message":"class \"authors\": conflict for property \"Article\": already in use or provided multiple times"}]}
```

For example, when using:
```ts
await client.schema
      .propertyCreator()
      .withClassName(className)
      .withProperty(property)
      .do()
      .catch((err: Error) => {
        console.warn(err);
      });
```

As a side note, I have created a simple CLI tool to update a class using a configuration file, [weaviate-class-updater](https://github.com/erikvullings/weaviate-class-updater/tree/main)

**Comment by erikvullings** [comment]
@tsmith023 Thanks for looking into it. As requested, I have moved the issue to [Weaviate](https://github.com/weaviate/weaviate/issues/3166#issue-1771367156).
